### ISSUE TITLE
Update website images

### ISSUE THREAD

**Issue by MagicLike** [openandroidinstaller-dev/openandroidinstaller-dev.github.io#15]
From my POV the images are no longer up-to-date and some are even no longer needed.

**Comment by tsterbak** [comment]
Which images do you mean? 🤔

Edit: A you mean the screenshots? Yeah, they could be removed.

**Comment by MagicLike** [comment]
> A you mean the screenshots? Yeah, they could be removed.

Yup I meant those - just clicked through the directories and did not know which were in use...
